php/actions/get_torrent_files.php

```diff
@@ -124,6 +128,8 @@
                         $tracker = preg_replace(['/(?<=\.)([-\w]+\.\w+)/', '/\w+(?==)/'], ['t-ru.org', 'pk'], $tracker);
                     }
                 }
+                $temp[] = 'after => ' . $tracker;
+                Log::append(implode(',', $temp));
```

Comment:
As the commit message says, don't forget to remove it! :)

Reply:
done